src/ApiService/ApiService/onefuzzlib/Utils.cs

```diff
@@ -15,3 +15,22 @@ public static class ObjectExtention {
     public static Async.Task IgnoreResult<T>(this Async.Task<T> task)
         => task;
 }
+
+public static class IAsyncEnumerableExtension {
+    public static async IAsyncEnumerable<List<TSource>> Chunk<TSource>(this IAsyncEnumerable<TSource> source, int size) {
+
+        var enumerator = source.GetAsyncEnumerator();
+        List<TSource> result = new List<TSource>(size);
+        while (await enumerator.MoveNextAsync()) {
+            if (result.Count == size) {
+                yield return result;
+                result = new List<TSource>(size);
+            }
+            result.Add(enumerator.Current);
```

Comment:
Do this in the other order so it `yield`s the result as soon as it is ready?

```suggestion
            result.Add(enumerator.Current);
            
            if (result.Count == size) {
                yield return result;
                result = new List<TSource>(size);
            }
```